webinar.html

```diff
@@ -0,0 +1,231 @@
+<li><strong>2020/9/4</strong>: <a href="https://www.youtube.com/watch?v=S27pHKBEp30&ab_channel=SeattleAppliedDeepLearning"> LSTM is dead. Long Live Transformers!   </a> </li>
+
+<h2>
+COVID Break
+</h2>
+
+<li><strong>2019/10/03</strong>: <a href="https://www.youtube.com/watch?v=8CenT_4HWyY">3D deep learning (CVPR 2017)</a> : 43:00 to 1:08:00</li>
+<li><strong>2019/09/19</strong>:&nbsp;</strong><a href="https://www.youtube.com/watch?v=Bo8MY4JpiXE">Interview with Francois Cholet.</a></li>
+<li><strong>2019/09/12</strong>:&nbsp;<a href="https://www.youtube.com/watch?v=_-RU9Yoca84">Capsule Networks for computer vision â CVPR 2019</a></li>
+<li><strong>2019/09/05</strong>: <a href="https://www.youtube.com/playlist?list=PLe5rNUydzV9QHe8VDStpU0o8Yp63OecdW">Introduction on deep Bayesian methods</a></li>
+<li><strong>2019/08/01:</strong>&nbsp;<a href="http://A new perspective on adversarial-perturbations">A new perspective on adversarial-perturbations</a></li>
+<li><strong>2019/07/11-18:</strong>&nbsp;<a href="https://www.youtube.com/watch?v=KDRN-FyyqK0">ICML 2018: Tutorial Session: Toward the Theoretical Understanding of Deep Learning</a></li>
+<li><strong>2019/07/04</strong>: <a href="https://www.youtube.com/watch?v=z1xs9jdZnuY">Attention is all you need</a></li>
+<li><strong>2019/06/27:</strong>&nbsp;<a href="https://cbmm.mit.edu/video/principles-and-applications-relational-inductive-biases-deep-learning">Principles and applications of relational inductive biases in deep learning</a>&nbsp;by&nbsp;Kelsey Allen</li>
+<li><strong>2019/06/13:</strong> <a href="https://www.youtube.com/watch?v=CrkGMnh7KiA">Active learning</a></li>
+<li><strong>2019/05/23-30 â 06/06</strong>:&nbsp; <a href="https://www.youtube.com/watch?time_continue=2333&amp;v=fQyT5wkFxGI">Adversarial robustness : theory and practice</a>. (NeuRIPS tutorial 2018)</li>
+<li><strong>2019/05/16</strong>:&nbsp; <a href="https://www.youtube.com/watch?v=TB07_mUMt0U">Deep learning determinism</a>. by&nbsp;Duncan Riach</li>
+<li><strong>2019/05/09</strong>:<a href="https://www.youtube.com/watch?v=e5othcNmync">&nbsp;Guarding user Privacy with Federated Learning and Differential Privacy </a>by B. McMahan</li>
+<li><strong>2019/05/02:</strong>&nbsp;<a href="https://www.youtube.com/watch?v=0eBR8a4MQ30">Automatic Machine Learning</a> (NeurIPS 2018 Turorial) part2</li>
+<li><strong>2019/05/02:</strong><a href="https://www.youtube.com/watch?v=0eBR8a4MQ30">&nbsp;Automatic Machine Learning</a> (NeurIPS 2018 Tutorial) part 1</li>
+<li><strong>2019/04/04</strong>: Introduction to the new âBelugaâ GPU server</li>
+<li><strong>2019/03/28</strong>: <a href="https://livestream.com/newyorkacademyofsciences/ml2019-3/videos/188158627">Integrating Domain-Knowledge into deep learning</a>&nbsp;by Ross Salakoudenov</li>
```

Comment:
Vimeo livestreams can be embedded like this:
```suggestion
<li><strong>2019/03/28</strong>: Integrating Domain-Knowledge into deep learning&nbsp;by Ross Salakoudenov</li>
<iframe id="ls_embed_1615822128" src="https://livestream.com/accounts/3261852/events/8571609/videos/188158627/player?width=560&height=315&enableInfo=true&defaultDrawer=&autoPlay=true&mute=false" width="560" height="315" frameborder="0" scrolling="no" allowfullscreen> </iframe>
```